tests/integration/targets/aci_access_span_filter_group_entry/tasks/main.yml

```
@@ -0,0 +1,250 @@
+# Test code for the ACI modules
+# Copyright: (c) 2023, Akini Ross (akinross@cisco.com)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: 'Please define the following variables: aci_hostname, aci_username and aci_password.'
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+- name: Set vars
+  ansible.builtin.set_fact:
+    aci_info: &aci_info
+      host: "{{ aci_hostname }}"
+      username: "{{ aci_username }}"
+      password: "{{ aci_password }}"
+      validate_certs: '{{ aci_validate_certs | default(false) }}'
+      use_ssl: '{{ aci_use_ssl | default(true) }}'
+      use_proxy: '{{ aci_use_proxy | default(true) }}'
+      output_level: '{{ aci_output_level | default("info") }}'
+
+- name: Verify Cloud and Non-Cloud Sites in use.
+  include_tasks: ../../../../../../integration/targets/aci_cloud_provider/tasks/main.yml
+
+- name: Execute tasks only for non-cloud sites
+  when:
+  - query_cloud.current == []
+  block:
+  
+  # CLEAN TEST ENVIRONMENT
+
+  - name: Query all access span filter groups
+    cisco.aci.aci_access_span_filter_group:
+      <<: *aci_info
+      state: query
+    register: query_for_clean
+
+  - name: Clean access span filter groups
+    cisco.aci.aci_access_span_filter_group:
+      <<: *aci_info
+      filter_group: "{{ item.spanFilterGrp.attributes.name }}"
+      state: absent
+    loop: "{{ query_for_clean.current }}"
+
+  - name: Add access span filter group
+    cisco.aci.aci_access_span_filter_group: &add_ansible_filter_group_1
+      <<: *aci_info
+      filter_group: ansible_group_1
+      state: present
+
+  - name: Add access span filter group
+    cisco.aci.aci_access_span_filter_group: &add_ansible_filter_group_2
+      <<: *aci_info
+      filter_group: ansible_group_2
+      state: present
+
+  # TEST CREATE FILTER GROUP ENTRIES
+
+  - name: Add access span filter group entry ( checkmode )
```

The task name is repeated multiple times. Please provide unique task names.
```suggestion
  - name: Add access span filter group entry 1 to ansible_group_1( checkmode )
```